index.d.ts

```diff
@@ -4,8 +4,8 @@ import { Operator } from 'rxjs/Operator';
 
 export declare class ActionsObservable extends Observable<Action> {
   constructor(actionsSubject: Observable<Action>);
-  lift(operator: Operator<any, Action>);
-  ofType(...key: any[]);
+  lift(operator: Operator<any, Action>) : ActionsObservable;
+  ofType(...key: any[]) : boolean;
```

Comment:
https://travis-ci.org/redux-observable/redux-observable/builds/152735720#L388

```
test/typings.ts(10,6): error TS2339: Property 'mapTo' does not exist on type 'boolean'.
```

`ofType` also returns `ActionsObservable`, not `boolean`. The boolean returns you see in [`ofType`](https://github.com/redux-observable/redux-observable/blob/master/src/ActionsObservable.js#L16-L30) are for returning from the `filter` predicate.